task5/task5.ipynb

```diff
@@ -126,30 +151,34 @@
     "import matplotlib.pyplot as plt\n",
     "\n",
     "# RUN ALL CELLS ABOVE BEFORE RUNNING THIS CELL\n",
-    "constant_rates_expected_lifetime = 5.29201491219346\n",
-    "changing_rates_expected_lifetime = 2.273317852896713\n",
-    "\n",
+    "constant_rates_expected_lifetime = 2.300437982687964\n",
+    "changing_rates_expected_lifetime = 1.5056818182033227\n",
     "\n",
     "\n",
-    "t = np.linspace(0,10,11)\n",
+    "t_const = np.arange(0,len(active_probs_constant_retention[:12]),1)\n",
+    "t_changing = np.arange(0,len(active_probs_changing_retention_cohort0),1)\n",
     "bar_width = 0.3\n",
+    "print(active_probs_changing_retention_cohort0)\n",
+    "print(active_probs_constant_retention)\n",
     "\n",
     "fig, axs = plt.subplots(2, 1, figsize=(12,12))\n",
     "\n",
-    "axs[0].bar(t-bar_width/2, active_probs_constant_retention[:11], bar_width, color='r', label=\"5a: Average\")\n",
-    "axs[0].bar(t+bar_width/2, active_probs_changing_retention[:11], bar_width, color='b', label=\"5b: Monthly changing rates\")\n",
+    "axs[0].bar(t_const-bar_width/2, active_probs_constant_retention[:12], bar_width, color='r', label=\"5a: Average\")\n",
```

Comment:
5c: updated plot with changing rates for cohort 0 only. added expectation value for constant rate for cohort 0.